apps/aechannel/src/aesc_fsm.erl

```diff
@@ -3849,6 +3851,27 @@ start_noise_session(_,
     lager:debug("COpts = ~p", [COpts]),
     aesc_session_noise:connect(Host, Port, NoiseOpts).
 
+maybe_log_error({error, _} = Error, #{ responder := Responder
+                                      , initiator := Initiator}) ->
+    lager:error("Failed with ~p, Initiator: ~p, Responder: ~",
```

Comment:
This function is not needed, the lager error call should go into the relevant function clause of `noise_accept/3`.

Reply:
I didn't want to polute the argument's set with the COpts. Will address